code/game/objects/items/weapons/storage/mre.dm

```diff
@@ -171,6 +171,20 @@ MRE Stuff
 /obj/item/storage/box/fancy/mre/menu11/special // antag item
 	meal_desc = "This one doesn't have a menu listing. How odd. It has the initials \"A.B.\" written on the back."
 
+/obj/item/storage/box/fancy/mre/menu12_kois
+	name = "k'ois MRE"
+	meal_desc = "This one is menu 12, k'ois. DO NOT CONSUME IF YOU ARE NOT VAURCA. Contains k'ois. Only safe for consumption by Vaurca."
```

Comment:
```suggestion
	meal_desc = "This one is menu 12, k'ois. DO NOT CONSUME IF YOU ARE NOT VAURCA."
```
redundant text

Reply:
It's redundant so it is more visible and highlighted that it has k'ois and it's not just a normal MRE

Reply:
You would achieve the same effect in a much better way by bolding it and/or making the text red

Reply:
done it as suggested